src/PresentationalComponents/Email/Email.js

```diff
@@ -1,11 +1,9 @@
 import React, { useState, useEffect, Fragment } from 'react';
 import './email.scss';
 import { useSelector, useDispatch } from 'react-redux';
-import { formFieldsMapper, layoutMapper } from '@data-driven-forms/pf4-component-mapper';
+import { componentMapper } from '@data-driven-forms/pf4-component-mapper';
```

Comment:
If you want to save some bytes, you can import the components one by one and leave the wizard out. I don't think that anybody should be using a wizard in the user preferences and its probably the biggest component of the whole mapper.

But you will have to pay attention to cjs/esm imports if you do that.

Reply:
We don't know that yet, I'd rather keep it here rather than going back and re-introduce it.